pkgs/top-level/python-packages.nix

```diff
@@ -7333,6 +7350,9 @@ in {
       webtest
       zope_component
       zope_interface
+      plaster
+      plaster-pastedeploy
+      hupper
```

Comment:
why are these added to `buildInputs`? Seems to be needed at runtime (install_requires).